### PR TITLE
Improve the webapp deployment approach 

### PR DESCRIPTION
**Pull request by bastiao**
As discussed in #63 added a better way to build the webapp. It allows to install node and npm, and also to build the webapp for production. 

For build you just need to run `mvn install`. Alternative, and you may want, skip the npm just do:

`mvn install -Dskip.npm`

### CODE REVIEW

**Comment by Enet4** [comment]
Well, as I recall from #63, I still uneasy for me that I would have to install a "pocket" version of Node.js and npm in order to use that script, even when it's already installed in my system. Ideally, we would have a way to detect whether supported versions of `node` and `npm` are available (maybe by calling `npm --version`) and use those instead.

On the other hand, I can simply choose to skip that step and build the webapp manually. I'm a bit torn, because even something as simple as a shell script would suit me.

One more remark: is the documentation properly documented? Step 3 of building Dicoogle still states that you must navigate to the webapp and run `npm install`.

**Comment by bastiao** [comment]
@Enet4 fully agree. We need to sustain the deployment. If you want to keep developing, skip this installation is just an easier way to go.
Just fixed the README.md, thank you.

File: README.md [hunk 1]
```diff
@@ -135,11 +135,10 @@ If you are in the development process and using the UI, you may need to change y
 
 
  1. Retrieve the full source code from this repository: `git clone https://github.com/bioinformatics-ua/dicoogle.git`
- 2. Navigate to the web app directory: "./dicoogle/src/main/resources/webapp/"
- 3. Build the web application: `npm install`
-    - The full instructions can be read [here](dicoogle/src/main/resources/webapp/README.md).
- 4. Navigate to the project's base directory, and build the parent Maven project by calling `mvn install`.
- 5. The resulting jar file can be found in "./dicoogle/target".
+ 2. Navigate to the project's base directory, and build the parent Maven project by calling `mvn install`.
+    - Note: if you want, you can skip the npm part: `mvn install -Dskip.npm`
```
Comment (Enet4) [inline README.md]: I think it's not clear that this part refers to building the webapp. Maybe we should either extend this note or add a second list of steps to do it manually.

Comment (bastiao) [inline README.md]: sure, that's well be clarified in the pending PR regarding README. Thank you

**Comment by tmgodinho** [comment]
I think that this is a cool approach if it works well under windows.

**Comment by bastiao** [comment]
it works in windows, indeed.

**Pull request merged**